tests/exporters/exporters_utils.py

```diff
@@ -0,0 +1,58 @@
+PYTORCH_EXPORT_MODELS_TINY = {
```

Comment:
I think this could be the occasion to get rid of the `PYTORCH_EXPORT_MODELS` in `test_onnx_export.py` and use this everywhere, wdty?

Reply:
Yes, I want to do it in a next PR where we'll have nightly tests for the ONNX export with various shapes, potentially a bit larger models, while the normal (on PR and main) exporters tests would use this `TINY` dict!